fetch-mods/fetch-mods.ts

```diff
@@ -116,8 +116,14 @@ export async function fetchMods(modsJson: string) {
     }
   }
 
-  function getCleanedUpRelease(release: OctokitRelease) {
-    const asset = release.assets[0];
+  function getCleanedUpRelease(release: OctokitRelease, uniqueName: string) {
+    var asset = release.assets[0];
+    if (uniqueName == "bbepis.BepInEx") {
+      const filteredAssets = release.assets.filter((asset) => asset.browser_download_url.includes("x86") && !asset.browser_download_url.includes("Legacy"))
```

Comment:
Do we need this whole mess here? Aren't you using your own branch for BepInEx? It's only gonna have the files you want, right?